closed/OpenJ9.gmk

```diff
@@ -326,9 +326,11 @@ endif # OPENJ9_ENABLE_CUDA
 ifeq (true,$(OPENJ9_ENABLE_INLINE_TYPES))
   CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_VALUE_TYPES=ON
   CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES=ON
+  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_STRICT_FIELDS=ON
 else # OPENJ9_ENABLE_INLINE_TYPES
   CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_VALUE_TYPES=OFF
   CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES=OFF
+  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_STRICT_FIELDS=OFF
 endif # OPENJ9_ENABLE_INLINE_TYPES
```

Comment:
Please sort these additions:
```
ifeq (true,$(OPENJ9_ENABLE_INLINE_TYPES))
  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES=ON
  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_STRICT_FIELDS=ON
  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_VALUE_TYPES=ON
else # OPENJ9_ENABLE_INLINE_TYPES
  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES=OFF
  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_STRICT_FIELDS=OFF
  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_VALUE_TYPES=OFF
endif # OPENJ9_ENABLE_INLINE_TYPES
```
This change should be proposed at https://github.com/ibmruntimes/openj9-openjdk-jdk (and possibly backported to earlier versions); it will then flow naturally into this repository.

Reply:
Please sort so there is no conflict here when merging with https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/1093.

Reply:
> The valuetypes repo is quite behind, I think it would make sense to also merge this one until its updated so the StrictFieldTests will pass in the Valhalla build. 

That's right, a PR merging latest OpenJDK/Valhalla content is in progress.